File: Triatlon-Java/ChatModel/src/main/java/triathlon/model/ParticipantDTO.java
package triathlon.model;

import java.io.Serializable;

public class ParticipantDTO implements Serializable {
    private Integer id;
    private String firstName;
    private String lastName;
    private Double score;

    public ParticipantDTO(Integer id, String firstName, String lastName, Double score) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.score = score;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public Double getScore() {
        return score;
    }

    public void setScore(Double score) {
        this.score = score;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }
}

File: Triatlon-CS/Model/ParticipantDTO.cs
using System;

namespace Triatlon_CSharp.Domain
{
    [Serializable]
    public class ParticipantDTO
    {
        public ParticipantDTO() {}
        
        public ParticipantDTO(int id, string firstName, string lastName, Double score)
        {
            Id = id;
            FirstName = firstName;
            LastName = lastName;
            Score = score;
        }

        public int Id { get; set; }
        
        public String FirstName { get; set; }

        public String LastName { get; set; }
        
        public Double Score { get; set; }

        public override string ToString()
        {
            return string.Format("Id = {0} | {1} {2} {3}", Id, FirstName, LastName, Score);
        }
    }
}